Library/Formula/llvm.rb

```diff
@@ -78,17 +105,23 @@ class Llvm < Formula
       url "http://llvm.org/git/lldb.git"
     end
 
-    resource "clang-tools-extra" do
-      url "http://llvm.org/git/clang-tools-extra.git"
+    resource "polly" do
+      url "http://llvm.org/git/polly.git"
     end
   end
 
+  keg_only :provided_by_osx
+
   option :universal
-  option "with-clang", "Build Clang support library"
+  option "with-clang", "Build the Clang compiler and support libraries"
+  option "with-clang-extra-tools", "Build extra tools for Clang"
+  option "with-compiler-rt", "Build Clang runtime support libraries for code sanitizers, builtins, and profiling"
+  option "with-libcxx", "Build the libc++ standard library"
   option "with-lld", "Build LLD linker"
   option "with-lldb", "Build LLDB debugger"
-  option "with-rtti", "Build with C++ RTTI"
+  option "with-polly", "Build with the experimental Polly optimizer"
```

Comment:
Ohhh, so *that*'s what you're supposed to do. I misinterpreted your comment and just got rid of the stable version of the resource, leaving the option where it was. I'll fix that...